ush/python/pygfs/task/upp.py

```diff
@@ -249,7 +249,7 @@ def _call_executable(exec_cmd: Executable) -> None:
     @logit(logger)
     def finalize(upp_run: Dict, upp_yaml: Dict) -> None:
         """Perform closing actions of the task.
-        Copy data back from the DATA/ directory to COM/
+        Copy data back from the DATA/ directory to COMIN/ or COMOUT/
```

Comment:
By definition, you do not write to `COMIN`, only `COMOUT`
```suggestion
        Copy data back from the DATA/ directory to COMOUT/
```

Reply:
@WalterKolczynski-NOAA 
Yes, I agree with your suggestion. I make the change.

Reply:
@WalterKolczynski-NOAA 
Yes, I have made change based on your suggestion.